test/unit/calculators/check_travel_during_coronavirus_calculator_test.rb

```diff
@@ -120,5 +120,51 @@ class CheckTravelDuringCoronavirusCalculatorTest < ActiveSupport::TestCase
       @calculator.countries = %w[spain]
       assert @calculator.single_journey?
     end
+
+    context "vaccination_options" do
+      should "return a hash of options" do
```

Comment:
I am wondering if this is too vague? or is it intended to be flexible incase the file changes? I'm wondering if we should use a test config file?

Reply:
👍